core/include/misc/uri.h

```diff
@@ -0,0 +1,50 @@
+//
+// Created by Jake Bolewski on 7/20/17.
+//
+
+#ifndef TILEDB_URI_H
+#define TILEDB_URI_H
+
+#include <string>
+#include "status.h"
+
+namespace tiledb {
+
+namespace uri {
```

Comment:
I think that if we have stand-alone functions, we will just make them static in class URI.